src/components/Album.jsx

```diff
@@ -1,19 +1,21 @@
-import { PropTypes } from 'prop-types';
-import { CoverImage } from './CoverImage';
+import { PropTypes } from "prop-types";
+import { AlbumName } from "./AlbumName";
+import { CoverImage } from "./CoverImage";
 
 export const Album = ({ album }) => {
   // TODO: remove this later on
-  console.log(album);
+  // console.log(album);
 
-  const image = album.images.find(image => image.height === 640);
+  const image = album.images.find((image) => image.height === 640);
 
   return (
     <div className="album-container">
-      <CoverImage img={image.url} />
-      {/* 
-        // AlbumName component here
-        // ArtistName component here  
-      */}
+      <CoverImage img={album.images[1].url} alt={album.name} />
+      {
+        <div>
```

Comment:
let's remove this extra div here since it's not doing anything.

Reply:
Done!